AdyenActions/UI/View Controllers/Voucher/VoucherView.swift

```diff
@@ -248,4 +247,16 @@ internal final class VoucherView: UIView, Localizable {
         )
     }
     
+    override public func didMoveToWindow() {
```

Comment:
What do you think of adding a protocol that all these views conform to and handle this duplicate code block in its extension?

Reply:
Good one! Ideally this logic actually does not happen inside of the view but in the viewController. For now I just moved the logic out of the NetworkImageView to the parent view to keep the scope small. I'll look into a better separation of concerns in a future PR.